transport/internal/safelycall.go

```diff
@@ -56,3 +56,21 @@ func SafelyCallHandler(h transport.Handler, start time.Time, ctx context.Context
 
 	return err
 }
+
+// SafelyCallRegistry calls procedures on the given registry safely.
```

Comment:
Please describe what you mean by "safe" in this case, like "rather than call directly and risk a panic in the absence of a serviceable handler for the service and procedure"